src/Utils/removeDuplicates.ts

```diff
@@ -0,0 +1,81 @@
+export const groupAndMergeByColumn = (
+  data: string,
+  groupByColumnIndex: number,
+  mergeColumnIndices: number[],
+  delimiter = ";",
+): string => {
+  if (!data || typeof data !== "string") {
+    throw new Error("Data is in Invalid Format");
+  }
+  const msg = "An error occurred while processing the export";
+
+  if (groupByColumnIndex < 0) {
+    throw new Error(msg);
+  }
+  if (!mergeColumnIndices.length) {
+    throw new Error(msg);
+  }
+  if (mergeColumnIndices.some((index) => index < 0)) {
+    throw new Error(msg);
+  }
+  const lines = data.split("\n"); // Split the data into individual lines
```

Comment:
won't the first row be the header? 🤔 

```suggestion
  const [header, ...data] = data.trim().split("\n");
```